e2e/live/end-to-end-test.sh

```diff
@@ -419,422 +419,423 @@ set +e                          # Do not stop the test for errors
 
 createTestSuite
 waitForDefaultServiceAccount
+# TODO(rquitales): Re-enable e2e tests once all functionality implemented.
```

Comment:
e2e tests are guarding kpt repo's healthiness. Once disabled, other PRs won't be able to get the proper test coverage.
Do we have other ways to not disable these tests? If not, you might need to coordinate with other contributors to make sure no PRs are coming up until those e2e tests are re-enabled (though I am not sure if it is realistic)